docs/guides/high-availability/introduction.adoc

```diff
@@ -12,6 +12,101 @@ Those setups are intended for a transparent network on a single site.
 The {project_name} high-availability guide goes one step further to describe setups across multiple sites.
 While this setup adds additional complexity, that extra amount of high availability may be needed for some environments.
 
+== When to use a multi-site setup
+
+The multi-site deployment capabilities of {project_name} are targeted at use cases that:
+
+* Are constrained to a single
+<@profile.ifProduct>
+AWS Region.
+</@profile.ifProduct>
+<@profile.ifCommunity>
+AWS Region or an equivalent low-latency setup.
+</@profile.ifCommunity>
+* Permit planned outages for maintenance.
+* Fit within a defined user and request count.
+* Can accept the impact of periodic outages.
+
+<@profile.ifCommunity>
+== Tested Configuration
+
+We regularly test {project_name} with the following configuration:
+</@profile.ifCommunity>
+<@profile.ifProduct>
+== Supported Configuration
+</@profile.ifProduct>
+
+* Two Openshift single-AZ clusters, in the same AWS Region
+** Provisioned with https://www.redhat.com/en/technologies/cloud-computing/openshift/aws[Red Hat OpenShift Service on AWS] (ROSA),
+<@profile.ifProduct>
+either ROSA HCP or ROSA classic.
+</@profile.ifProduct>
+<@profile.ifCommunity>
+using ROSA HCP.
+</@profile.ifCommunity>
+
+** Each Openshift cluster has all its workers in a single Availability Zone.
+** OpenShift version
+<@profile.ifProduct>
+4.16 (or later).
+</@profile.ifProduct>
+<@profile.ifCommunity>
+4.16.
```

Comment:
Do we need to differentiate here, it seems like it's safe to say "4.16 (or later)" for community as well given that we need to verify this for the product.

Reply:
The wording is related to the heading. For downstream, we say "we support 4.16 (or later)". For upstream it is "we test 4.16". Writing "we test 4.16 (or later)" would be a bit unspecific on what we are testing, so this is why I have the two versions. Does that make sense?

Reply:
Yes that makes sense :+1: 

Tbh it's very hard to read this file with all of the `if/else` logic. Including two different files with product/community content would make things much easier to follow, although this obviously comes at the expense of some duplicated content.